browser-tests/childrenFeature.ts

```diff
@@ -117,8 +120,8 @@ test('As a guardian I want to add and delete a child', async (t) => {
   await t.wait(1000); // 1s
 
   // Assert that the child can be found
-  await t.expect(godchildrenProfilePage.child(newChildName).exists).ok();
+  await t.expect(godchildrenProfilePage.child(newChildNameRegepx).exists).ok();
```

Comment:
This line causes browser test to fail.

Reply:
Hmn, bit of an unfortunate issue because it seems that it has to do with the wait time. The screenshot of the original error contained the button for the created child. I re-ran the suite now and it passed. My guess is that not a lot of extra time is needed, so I added 0.5s.